app/controllers/users_controller.rb

```diff
@@ -1,4 +1,30 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  def show
+    @user = User.find_by(id: params[:id])
+    return if @user
+
+    flash[:warning] = t("not_found_user")
+    redirect_to(root_path)
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to(user_path(id: @user.id))
```

Comment:
cho người ta cái mess để bjk thành công chứ
cơ mà redirect dài thế 
```suggestion
      redirect_to @user
```
ri cho hịn